pyproject.toml

```diff
@@ -1,2 +1,9 @@
 [tool.black]
 line-length = 79
+
+[tool.setuptools_scm]
+version_scheme = "post-release"
+
+[build-system]
```

Comment:
Do we need this? I thought it was only needed if `pyproject.toml` was being used by `pip` to build the package rather than `setup.py`?

Reply:
@liamcoatman it is a good challenge. I re-read their recommendations' meanings, and tried install without this part. I'm _pretty confident_ now that indeed it's not needed in the current setup, so pushed an extra change with this removed.